### PR TITLE
Fix/setup.cfg

### LINKED ISSUE #154: pin tifffile dependency to something greater than version 2023.*
Using a tifffile version from 2021 was incompatible with tiffslide. To be on the safe side, let's pin the tifffile version to something equal to or newer than 2023.*.

The change would have to be made in `setup.cfg`.

### PR DESCRIPTION
**Pull request by swaradgat19**
Pinned imagecodecs, tiffslide and tifffile in setup.cfg

fixes #154

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -38,8 +38,9 @@ install_requires =
     pyyaml
     scikit-image>=0.20.0
     shapely
-    tifffile
-    tiffslide
+    tifffile>=2023.1.23
+    tiffslide>=2.2.0
```
Comment (kaczmarj) [inline setup.cfg:42]:
```suggestion
    # https://github.com/Bayer-Group/tiffslide/issues/72#issuecomment-1631015274
    tiffslide>=2.2.0
```

File: setup.cfg [hunk 2]
```diff
@@ -38,8 +38,9 @@ install_requires =
...
+    imagecodecs >= 2023.7.10  # for tifffile
```
Comment (kaczmarj) [inline setup.cfg:43]:
```suggestion
    # https://github.com/Bayer-Group/tiffslide/issues/72#issuecomment-1630091390
    imagecodecs >= 2023.7.10  # for tifffile
```

**Review by kaczmarj — CHANGES_REQUESTED** [review]

**Review by kaczmarj — APPROVED** [review]

**Comment by kaczmarj** [comment]
@swaradgat19 - thanks! looks great. i added two comments to the setup.cfg so we know why these packages are pinned to those versions.

the python 3.8 test is failing because the latest version of imagecodecs isn't built for python 3.8.

i think adding this will fix it...

```
imagecodecs >= 2022.7.27 ; python_version<3.9
imagecodecs >= 2023.7.10 ; python_version>=3.9
```

**Comment by swaradgat19** [comment]
@kaczmarj Right. I'll do the changes.

**Comment by swaradgat19** [comment]
The rest of them are canceled since the 3.8 test didn't pass right?

**Comment by kaczmarj** [comment]
some tests passed, two tests are still running. the tests across python 3.8-3.11 all stopped because 3.8 failed.

**Comment by swaradgat19** [comment]
Yeah I meant the 3.9-3.11 ones. Got it

**Comment by swaradgat19** [comment]
@kaczmarj Added the comments as well. Does the syntax go `imagecodecs >= 2022.7.27 ;python_version<"3.9"` , with the quotes. Saw it here [link](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html). Let me know if changes are required.

**Comment by kaczmarj** [comment]
@swaradgat19  - yes, looks like the quotes around the python version are important. can you make that change?

**Comment by swaradgat19** [comment]
@kaczmarj Yes. I've pushed the changes

**Pull request merged**